nautobot/extras/tests/test_filters.py

```diff
@@ -1144,6 +1147,11 @@ def test_job_model(self):
                 self.filterset(params, self.queryset).qs, self.queryset.filter(job_model__in=jobs).distinct()
             )
 
+    def test_scheduled_job(self):
+        job_results = list(JobResult.objects.all()[:3])
+        scheduled_jobs = list(ScheduledJob.objects.all())
+        self.assertEqual(scheduled_jobs[0], job_results[1].scheduled_job)
```

Comment:
FWIW this isn't actually testing the `scheduled_job` FilterSet filter. To do that, the pattern to follow would be something more like the `test_job_model` test case above.

Reply:
I rewrote the test to be just like `test_job_model` above it